internal/compiler/widgets/fluent-base/scrollview.slint

```diff
@@ -33,6 +33,8 @@ component ScrollViewButton inherits TouchArea {
 }
 
 component ScrollBar inherits Rectangle {
+    callback scroll-event <=> i-touch-area.scroll-event;
```

Comment:
Is this still needed?

Reply:
No, thank you for the hint. It's removed now.